Detectors/TRD/reconstruction/src/DataReader.cxx

```diff
@@ -90,9 +91,14 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
   int idevice = 0;
   //  LOG(info) << "expected incoming data definition : " << inputspec;
   // this is probably never going to be used but would to nice to know hence here.
+  auto orig = o2::header::gDataOriginTRD;
```

Comment:
Does the reader supposed to handle other raw data than of RAWDATA type? You still have https://github.com/AliceO2Group/AliceO2/blob/cdb1d5b6df8c12a846e2c20a180e6a2049d809f1/Detectors/TRD/reconstruction/src/DataReader.cxx#L32 defined, but now it is ignored in favour of the hardcoded `RAWDATA`. If you will have only `RAWDATA`, then the `inputspec` should be removed. In case you need this flexibility in choosing the DataDescription, than you have to apply the trick which MCH implemented: https://github.com/AliceO2Group/AliceO2/blob/922cac2b59c7ea594baf17ef790972ed54dd6684/Detectors/MUON/MCH/Workflow/src/DataDecoderSpec.cxx#L291-L296 in https://github.com/AliceO2Group/AliceO2/pull/6512

Reply:
When I wrote that I got annoyed so just hardcoded it in(something was not working, cant remember what), not thinking that it needs compressed data as well. I would have bumped into that check this afternoon, so thanks for the catch.

Reply:
done